server/resources/available.toggles

```diff
@@ -19,7 +19,7 @@
       {
          "key": "browser_console_log_ws_key",
          "description": "Enables rendering console logs via websockets. Default: disabled.",
-         "value": false
+         "value": true
```

Comment:
Set the default value of `browser_console_log_ws_key` to false
Change the key to `build_details_page_ws_key` as it now handles jobStatus and console logs over websockets